meta-sokol-flex-distro/conf/distro/sokol-flex.conf

```diff
@@ -347,3 +350,6 @@ INITRAMFS_IMAGE ?= "flex-initramfs-image"
 INITRAMFS_IMAGE_BUNDLE ?= "${@bb.utils.contains('KERNEL_IMAGETYPES', 'fitImage', '', '1', d)}"
 ## }}}1
 # vim: set fdm=marker fdl=0 :
+
+# We have it here because we use upstream virtualization layer without any changes
+EXTRA_IMAGE_FEATURES += "${@bb.utils.contains('DISTRO_FEATURES', 'virtualization', 'virtualization', '', d)}"
```

Comment:
@aouahm0d first we should try if we can use feature override here. i.e.
```
EXTRA_IMAGE_FEATURES:append:feature-virtualization = " virtualization"
```
if that doesn't work we can use bb.utils.filter here.

Reply:
I have checked it. It does not work. So resorting to the already comitted bb.utils.filter